README.md

```diff
@@ -24,3 +24,10 @@ This repository contains templates for bootstrapping a Rust
    - [Simple](./simple/README.md)
    - [Simple Async](./simple-async/README.md)
    - [Component](./component/README.md)
+
+## Contributing
```

Comment:
Why "contributing" here? Another title might be more appropriate.

Reply:
Mostly habit - I usually have a section like this in various repos and slap any random tips for building / maintaining a repo in there.